docs/_static/gallery.yaml

```diff
@@ -1,16 +1,17 @@
-
 - title: Arxiv Paper Search
-  website: https://docsearch.redisvl.com
-  img-bottom: ../_static/gallery-images/arxiv-search.png
+  website: https://github.com/redis-developer/redis-arxiv-search
+  # img-bottom: ../_static/gallery-images/arxiv-search.png
```

Comment:
hanging comment here can we delete or does it belong?

Reply:
sure I can delete